Cinema Journal/Model/Networking.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Networking.swift
+//  Cinema Journal
+//
+//  Created by Homer on 10/21/20.
+//
+
+import Foundation
+
+struct Networking {
+    
+    static func fetchData() {
```

Comment:
Now you need to make that method more useful. 
Create some convenient  data source for endpoints (paths). Personally, I prefer enum-based : https://gist.github.com/iKisliy/889ba6b9d25cc246c540be8876598306
Then pass case to method